src/course-libraries/ReviewTabContent.tsx

```diff
@@ -227,6 +235,7 @@ const ComponentReviewList = ({
         <BlockCard
           key={info.usageKey}
           info={info}
+          libraryName={outOfSyncComponentsByKey[info.usageKey]?.upstreamContextTitle || ''}
```

Comment:
```suggestion
          libraryName={outOfSyncComponentsByKey[info.usageKey]?.upstreamContextTitle}
```

Very minor nit: if you declared `libraryName` as `?: string`, then it can be `undefined` so you don't need ` || ''`. Conversely, if you want to use `|| ''` then you don't have to make `libraryName` an optional prop with `?`.

Reply:
done 👍